extensions/pyRevitTools.extension/pyRevit.tab/Drawing Set.panel/views.stack/Legends.pulldown/Copy Legends to Other Documents.pushbutton/script.py

```diff
@@ -96,3 +114,44 @@ def OnDuplicateTypeNamesFound(self, args):
                         'Legend already exists. Renaming to: "%s"', new_name)
                 revit.update.set_name(dest_view, new_name)
                 dest_view.Scale = src_legend.Scale
+
+                # matching reference planes
+                for src_plane in reference_planes:
+                    src_param = \
+                        src_plane.GetParameter(
+                            DB.ParameterTypeId.ClineSubcategory)
+                    src_subcat = \
+                        DB.Category.GetCategory(
+                            revit.doc, src_param.AsElementId())
+                    dest_plane = \
+                        dest_doc.Create.NewReferencePlane(
+                            src_plane.BubbleEnd, 
+                            src_plane.FreeEnd, 
+                            DB.XYZ(0,0,1), 
+                            dest_view)
+                    if src_subcat.Id != src_plane_category.Id:
```

Comment:
to prevent too much indentation, and since it does nothing if the id matches, you can invert the condition and skip to the next loop iteration

```
if src_subcat.Id == src_plane_category.Id:
    continue
```